database/migrations/13_addApplicationAndEnvironmentColumnsToVersionsTable.go

```diff
@@ -9,14 +9,14 @@ func addApplicationAndEnvironmentColumnsToVersionsTable() *migrate.Migration {
 		Id: "13",
 		Up: []string{`
       ALTER TABLE versions
-	  	  ADD COLUMN application_id int(10) unsigned NOT NULL AFTER replicas,
-		    ADD COLUMN environment_uuid varchar(36) NOT NULL AFTER environment_id
+        ADD COLUMN application_id int(10) unsigned NOT NULL AFTER replicas,
```

Comment:
Will this fail if there are multiple versions or just put a non-null default value?